src/main/scala/gitbucket/core/api/CreateARef.scala

```diff
@@ -0,0 +1,7 @@
+package gitbucket.core.api
+
+/**
+ * https://developer.github.com/v3/git/refs/#create-a-reference
```

Comment:
Links to GitHub docs should point to https://docs.github.com/. Though, there are still many old links in GitBucket codebase, we are updating these links when we touch code around them one by one. Could you update some links which is covered by this pull request?

Reply:
Yeah no problem, I fixed the links.